Structure_oM/Versioning_73.json

```
@@ -0,0 +1,27 @@
+﻿{
+    "Namespace": {
+        "ToNew": {
+        },
+    "ToOld": {
+        }
+    },
+  "Type": {
+        "ToNew": {
+
+        },
+    "ToOld": {
+        }
+    },
+  "Property": {
+    "ToNew": {
+      "BH.oM.Structure.Elements.PadFoundation.TopOutline" :  "Perimeter"
+    },
+    "ToOld": {
+      "Perimeter": "BH.oM.Structure.Elements.PadFoundation.TopOutline"
```

Think the ToNew works as is, and the ToOld is not really used, but think this should be the principle used, to keep things a bit cleaner.
```suggestion
      "BH.oM.Structure.Elements.PadFoundation.TopOutline" :  "BH.oM.Structure.Elements.PadFoundation.Perimeter"
    },
    "ToOld": {
      "BH.oM.Structure.Elements.PadFoundation.Perimeter": "BH.oM.Structure.Elements.PadFoundation.TopOutline"
```